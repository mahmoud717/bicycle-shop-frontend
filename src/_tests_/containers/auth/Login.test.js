import React from 'react';

import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import Login from '../../../containers/auth/login';

const mockStore = configureMockStore();
const store = mockStore({});
const setup = () => shallow(
  <Provider store={store}>
    <Login />
  </Provider>,
);

describe('Login Component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });
  test('renders the bicycle list Component', () => {
    const page = component.find('.bicycle-list-container');
    expect(page).toMatchSnapshot();
  });
});
