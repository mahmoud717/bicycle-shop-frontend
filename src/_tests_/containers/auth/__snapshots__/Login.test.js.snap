// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component renders the bicycle list Component 1`] = `ShallowWrapper {}`;
